engine/flink/kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/kafka/source/flink/FlinkKafkaSource.scala

```diff
@@ -145,6 +205,16 @@ class FlinkKafkaSource[T](
 
 }
 
+object FlinkKafkaSource {
+  val OFFSET_RESET_STRATEGY_PARAM_NAME: ParameterName = ParameterName("offsetResetStrategy")
+
+  object OffsetResetStrategy extends Enumeration {
+    type OffsetResetStrategy = Value
+    val Continue, Reset, Restart = Value
+  }
```

Comment:
Can it be changed to Enumeratum? Unfortunately, a Scala Enumeration type does not support exhaustive pattern matching